Fix:
grader-tf.c:108:9: error: implicitly declaring library function 'log' with type 'double (double)' [-Werror,-Wimplicit-function-declaration]
--- src/grader-tf.c.orig	2003-11-26 01:47:48.000000000 -0600
+++ src/grader-tf.c	2022-01-26 00:13:31.000000000 -0600
@@ -18,6 +18,7 @@
  *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
